### PR TITLE
Feature/totp newest

### PR DESCRIPTION
**Pull request by bporter125**

### CODE REVIEW

**Review by jnpblrdrgz — APPROVED** [review]

File: src/iovation.LaunchKey.Sdk/Client/IServiceClient.cs [hunk 1]
```diff
@@ -65,5 +65,12 @@ public interface IServiceClient : IWebhookHandler
         /// </summary>
         /// <param name="user">The username or directory user ID to end the session for</param>
         void SessionEnd(string user);
+        
+        /// <summary>
+        /// Verifies a given TOTP is valid for a given user.
+        /// </summary>
+        /// <param name="user">LaunchKey Username, User Push ID, or Directory User ID for the End User</param>
+        /// <param name="otp">6-8 digit OTP code for to verify</param>
```
Comment (lordkyzr) [inline src/iovation.LaunchKey.Sdk/Client/IServiceClient.cs:73]: Typo here, maybe you meant "6-8 digit OTP code for verification"

File: src/iovation.LaunchKey.Sdk/Client/IDirectoryClient.cs [hunk 2]
```diff
@@ -40,5 +40,27 @@ public interface IDirectoryClient : IServiceManagingClient, IWebhookHandler
...
+        ///     well as managing TOTP.
+        /// </param>
+        DirectoryUserTotp GenerateUserTotp(string userId);
+        
+        /// <summary>
+        /// Removes a TOTP configuration from a given user.
+        /// </summary>
+        /// <param name="userId">
+        ///     Unique value identifying the End User in the your system. This value was used to create the Directory
+        ///     User and Link Device
```
Comment (lordkyzr) [inline src/iovation.LaunchKey.Sdk/Client/IDirectoryClient.cs:62]: isnt totp separate from directory users?

**Review by lordkyzr — APPROVED** [review]

File: src/iovation.LaunchKey.Sdk/Client/IDirectoryClient.cs (hunk 2 above)
Comment (bporter125) [reply src/iovation.LaunchKey.Sdk/Client/IDirectoryClient.cs:62]: Correct. I just removed that note.

File: src/iovation.LaunchKey.Sdk/Client/IServiceClient.cs (hunk 1 above)
Comment (bporter125) [reply src/iovation.LaunchKey.Sdk/Client/IServiceClient.cs:73]: Updated

**Review by lordkyzr — APPROVED** [review]

**Review by CMoncur — APPROVED** [review]

**Pull request merged**